lib/gollum/app.rb

```diff
@@ -683,12 +683,12 @@ def wiki_new
     #     :email     - The String email address.
     # message is sourced from the incoming request parameters
     # author details are sourced from the session, to be populated by rack middleware ahead of us
-    def commit_message
+    def commit_options
       msg               = (params[:message].nil? or params[:message].empty?) ? "[no message]" : params[:message]
-      commit_message    = { :message => msg }
+      commit_options    = { message: msg, notes: session['gollum.notes'] }
```

Comment:
What happens when `session['gollum.notes']` is not set? Will `notes` be `nil` or will this raise a KeyError of sorts?

Reply:
Just tested, it's `nil` as intended 👍 .